x-pack/plugins/security/server/session_management/session.ts

```diff
@@ -56,6 +56,12 @@ export interface SessionValue {
    */
   state: unknown;
 
+  /**
+   * Unique identifier of the user profile, if any. Not all users that have session will have an associated user
+   * profile, e.g. anonymous users won't have it.
+   */
+  userProfileUid?: string;
```

Comment:
**nit**: I find this a bit of a mouthful. The "u" in the ES response isn't really adding anything. It's kind of obvious that an identifier is unique. Maybe `profileId` or `userProfileId` would be a bit simpler while still being explicit about its meaning.